test/QuestFactory.t.sol

```diff
@@ -306,8 +305,7 @@ contract TestQuestFactory is Test, Errors, Events, TestUtils {
     }
 
     function test_claimCompressed_revert_txOriginMismatch() public{
```

Comment:
Should we just remove the whole test?

Reply:
Yeah, I always feel iffy about deleting tests that assert specific behavior like this. Open to removing it.

Reply:
Well we arn't asserting that behavior anymore, right?